tests/index.scss

```diff
@@ -0,0 +1,13 @@
+$use-embedded-font: true;
```

Comment:
I would also get rid of the styles. effectively removing one layer of complexity for the tests. For unit tests, we don't need them.
If we would do screenshot tests, this is a different thing.